Dockerfile

```
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.18 as builder
+FROM docker.io/library/golang:1.18 as builder
```

```suggestion
FROM quay.io/projectquay/golang:1.18 as builder
```
This would prevent the break from docker rate-limiting.